libs/geoservices/covid/src/lib/pages/maps/components/time-map/time-map.component.scss

```diff
@@ -0,0 +1,101 @@
+@import 'libs/sass/mixins';
+
+#map-section {
+  width: auto;
+  height: 50rem;
+  display: flex;
+
+  #map-section-description,
+  #map-section-map {
+    @include flexbox();
+    @include flex(1);
+    @include flex-shrink(10);
+  }
+
+  #map-section-description {
+    padding: 10rem 0;
+
+    p {
+      max-width: 50%;
+    }
+  }
+}
+
+#map-section h3 {
+  padding: 1rem;
+}
+
+//legend start
+.legend {
+  background-color: #fff;
+  border-radius: 3px;
+  bottom: 7rem;
+  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.1);
+  padding: 10px;
+  position: absolute;
+  z-index: 1;
+}
```

Comment:
Legend is still positioned absolutely relative to the viewport (overlayed on top of the footer).

Add `position: relative;` to the right sidebar place it relative to the sidebar.

Reply:
added